tools/install.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
 set -e
 ## This script installs the Terra CLI from an unarchived release directory.
-## Dependencies: docker, gcloud
+## TERRA_CLI_DOCKER_MODE environment variable controls docker support. Set to
+##     DOCKER_NOT_AVAILABLE (default) to skip pulling the Docker image
+#      or DOCKER_AVAILABLE to pull the image (requires Docker to be installed and running).
```

Comment:
Please add notes about this env var and the default to the `README`, in the Install & run and Requirements sections.